src/main/java/com/marmelab/quoridor/web/BoardView.java

```diff
@@ -1,11 +1,11 @@
 package com.marmelab.quoridor.web;
 
-import com.marmelab.quoridor.model.Fence;
+import com.marmelab.quoridor.model.*;
```

Comment:
I don't like "import-it-all" things. You'll import every new class within that package?

Reply:
In general: explicit > implicit